lib/controllers/user_controller.dart

```diff
@@ -1 +1,30 @@
+import 'package:sqfentity_gen/sqfentity_gen.dart';
+import 'package:sun_flutter_capstone/models/model.dart';
 
+class UserController {
+  getAccount(String firstName, String lastName) async {
```

Comment:
For getting account, since there will be only one user, I think we can do this:
```
Future<Account?> getAccount() async {
    Account? account = await Account().select().toSingle();
    return account;
  }
```